playkit/src/main/java/com/kaltura/playkit/player/PlayerController.java

```diff
@@ -46,6 +46,8 @@
     private PKRequestParams.Adapter contentRequestAdapter;
 
     private boolean isNewEntry = true;
+    private boolean enableCea608 = false;
```

Comment:
This field should be named `cea608CaptionsEnabled`. The setter should be named `setCea608CaptionsEnabled`.